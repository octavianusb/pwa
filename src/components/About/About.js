import React from 'react'

import styles from './About.css'

export default function About () {
  return (
    <div>
      <h1 className={styles.heading}>{'Welcome to ABOUT page!'}</h1>
      <p className={styles.bio}>{'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.'}</p>
    </div>
  )
}
