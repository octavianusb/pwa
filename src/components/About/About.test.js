import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import About from './About'

describe('About component', () => {
  it('should render as expected', () => {
    const component = shallow(<About />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})