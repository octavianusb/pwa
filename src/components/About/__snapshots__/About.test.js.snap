// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should render as expected 1`] = `
<div>
  <h1
    className="heading"
  >
    Welcome to ABOUT page!
  </h1>
  <p
    className="bio"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
  </p>
</div>
`;
